src/components/content/activities/fill-in-the-blank.tsx

```diff
@@ -305,14 +351,23 @@ export function FillInTheBlankActivity({
                     onDrop={onSelectOption}
                     onClick={() => onDeselectOption(blank)}
                     selectedOption={getSelectedOption(blank)}
+                    isPrimary={props.textColorStyle === "primary"}
+                    canShowWrongAnswer
                   />
                 );
               }
             })}
           </p>
 
           {props.subText && (
-            <p className="mt-2 text-4xl leading-snug text-neutral-dark-700">
+            <p
+              className={clsx(
+                "mt-2 text-4xl leading-snug",
+                props.textColorStyle === "dark" && "text-neutral-dark-700",
+                props.textColorStyle === "light" && "text-neutral-100",
+                props.textColorStyle === "primary" && "text-primary-100",
+              )}
```

Comment:
This is repeated several times, maybe you can make a variable `textColor` that is assigned in a **switch** statement and then just use it everywhere in the file.
```suggestion
              className={clsx(
                "mt-2 text-4xl leading-snug",
                textColor,
              )}
```

Reply:
I have push a refactor commit to address this